packages/devtools_app/lib/src/app.dart

```diff
@@ -525,6 +528,67 @@ class SettingsDialog extends StatelessWidget {
   }
 }
 
+class _VerboseLoggingSetting extends StatelessWidget {
+  const _VerboseLoggingSetting();
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.only(
+        right: 6.0,
+        bottom: 6.0,
+      ),
+      decoration: const BoxDecoration(
+        border: Border.fromBorderSide(
+          BorderSide(width: 0.5, color: Colors.grey),
+        ),
+      ),
+      child: Column(
+        children: [
+          Row(
+            children: [
+              CheckboxSetting(
+                label: const Text('Enable verbose logging'),
+                listenable: preferences.verboseLoggingEnabled,
+                toggle: (enable) => preferences.toggleVerboseLogging(enable),
+                gaItem: gac.verboseLogging,
+              ),
+              const SizedBox(width: defaultSpacing),
+              IconLabelButton(
+                label: 'Copy logs',
+                icon: Icons.copy,
+                onPressed: () async => await copyToClipboard(
+                  LogStorage.root.toString(),
+                  'Successfully copied logs',
+                ),
+              ),
+              const SizedBox(width: defaultSpacing),
+              ClearButton(
+                label: 'Clear logs',
+                onPressed: LogStorage.root.clear,
+              ),
+            ],
+          ),
+          Container(
+            alignment: Alignment.center,
+            child: const Row(
+              mainAxisSize: MainAxisSize.min,
+              children: [
```

Comment:
can you add a todo here for me?
`TODO(kenz): style this with warning color once material 3 changes land`